Make.inc

```diff
@@ -220,6 +220,11 @@ includedir_rel := $(shell $(JULIAHOME)/contrib/relative_path.sh $(bindir) $(incl
 INSTALL_F := $(JULIAHOME)/contrib/install.sh 644
 INSTALL_M := $(JULIAHOME)/contrib/install.sh 755
 
+# BinaryBuilder options
+# TODO: Autodiscover triplet
```

Comment:
I think `$(shell $(CC) -dumpmachine)` should work in the majority of cases, with two exceptions: cross-compiling (in which case we already know the target triplet, right?) and platforms such as macOS and FreeBSD, where the minimum version number we support is part of the triplet. But if BinaryBuilder does triplet mapping using the `Platform` type, the latter case shouldn't be an issue, I would think.

Reply:
Yeah it is very close, but not quite and I don't want to spend time on figuring out the precise differences. So for now this has to be a conscious choice.